aries-site/src/themes/aries.js

```diff
@@ -204,6 +212,18 @@ export const aries = deepMerge(hpe, {
       xxlarge: '166px',
     },
   },
+  tab: {
+    border: {
+      color: 'text-xweak',
+    },
+    color: 'text-xweak',
+    margin: {
+      horizontal: 'none',
+    },
+    pad: {
+      horizontal: 'small',
```

Comment:
Assuming we are using this rather than `tabs.gap` because gap creates breaks in the border instead of the continuous line as shown in mock?

Reply:
Design looks like it is using a larger spacing between tabs, but bumping up forces tab to new row...  

Also, wondering how we translate this to mobile...

Reply:
Precisely. But setting "tabs.gap" to 'none' doesn't remove the gap. The horizontal margin needs to be removed on individual tabs to get the continuous line.